contents/blog/open-source-eating-saas.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2020-08-17
+title: Open source is eating SaaS
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Open source has several key advantages of traditional Software-As-A-Service (SaaS) business models. We think most companies building software will launch this way in the future.
```

Comment:
Depends on what the aim is - but maybe a more engaging first line could be good.